{"expand":"renderedFields,names,schema,operations,editmeta,changelog,versionedRepresentations","id":"40044","self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40044","key":"EUCA-11852","fields":{"issuetype":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issuetype/1","id":"1","description":"A problem which impairs or prevents the functions of the product.","iconUrl":"https://eucalyptus.atlassian.net/secure/viewavatar?size=xsmall&avatarId=14803&avatarType=issuetype","name":"Bug","subtask":false,"avatarId":14803},"timespent":null,"customfield_13100":null,"project":{"self":"https://eucalyptus.atlassian.net/rest/api/2/project/10000","id":"10000","key":"EUCA","name":"Eucalyptus","avatarUrls":{"48x48":"https://eucalyptus.atlassian.net/secure/projectavatar?pid=10000&avatarId=10011","24x24":"https://eucalyptus.atlassian.net/secure/projectavatar?size=small&pid=10000&avatarId=10011","16x16":"https://eucalyptus.atlassian.net/secure/projectavatar?size=xsmall&pid=10000&avatarId=10011","32x32":"https://eucalyptus.atlassian.net/secure/projectavatar?size=medium&pid=10000&avatarId=10011"}},"customfield_11000":null,"customfield_11001":null,"fixVersions":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/version/17200","id":"17200","description":"Eucalyptus 4.2.2","name":"4.2.2","archived":false,"released":true,"releaseDate":"2016-04-28"}],"aggregatetimespent":null,"resolution":{"self":"https://eucalyptus.atlassian.net/rest/api/2/resolution/5","id":"5","description":"All attempts at reproducing this issue failed, or not enough information was available to reproduce the issue. Reading the code produces no clues as to why this behavior would occur. If more information appears later, please reopen the issue.","name":"Cannot Reproduce"},"customfield_10310":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/10209","value":"CentOS 6","id":"10209"}],"customfield_11401":null,"customfield_10311":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/12401","value":"VPC-MIDO","id":"12401"}],"customfield_11400":null,"customfield_10302":null,"customfield_10104":[],"customfield_10500":{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/10403","value":"Not Applicable","id":"10403"},"customfield_10303":null,"customfield_10105":{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/10101","value":"No","id":"10101"},"customfield_10700":null,"customfield_10304":null,"customfield_10701":null,"customfield_10306":null,"customfield_10702":null,"customfield_10703":null,"customfield_10308":null,"customfield_10704":null,"resolutiondate":"2016-03-03T18:41:15.207-0600","customfield_10705":null,"customfield_10309":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/10204","value":"KVM","id":"10204"}],"workratio":-1,"lastViewed":null,"watches":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/EUCA-11852/watchers","watchCount":3,"isWatching":false},"created":"2015-12-20T19:33:40.476-0600","customfield_12000":null,"customfield_12400":null,"priority":{"self":"https://eucalyptus.atlassian.net/rest/api/2/priority/2","iconUrl":"https://eucalyptus.atlassian.net/images/icons/priorities/critical.svg","name":"Critical","id":"2"},"customfield_12201":null,"customfield_10102":null,"customfield_10300":null,"customfield_12600":"{}","customfield_10301":null,"labels":[],"customfield_11700":null,"aggregatetimeoriginalestimate":null,"timeestimate":null,"versions":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/version/16401","id":"16401","description":"Eucalyptus 4.2.1","name":"4.2.1","archived":false,"released":true,"releaseDate":"2015-12-07"}],"customfield_11901":null,"issuelinks":[{"id":"34719","self":"https://eucalyptus.atlassian.net/rest/api/2/issueLink/34719","type":{"id":"10103","name":"Relates","inward":"relates to","outward":"relates to","self":"https://eucalyptus.atlassian.net/rest/api/2/issueLinkType/10103"},"inwardIssue":{"id":"38608","key":"EUCA-11457","self":"https://eucalyptus.atlassian.net/rest/api/2/issue/38608","fields":{"summary":"EC2 instances stuck in pending","status":{"self":"https://eucalyptus.atlassian.net/rest/api/2/status/6","description":"The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.","iconUrl":"https://eucalyptus.atlassian.net/images/icons/statuses/generic.png","name":"Closed","id":"6","statusCategory":{"self":"https://eucalyptus.atlassian.net/rest/api/2/statuscategory/3","id":3,"key":"done","colorName":"green","name":"Done"}},"priority":{"self":"https://eucalyptus.atlassian.net/rest/api/2/priority/3","iconUrl":"https://eucalyptus.atlassian.net/images/icons/priorities/major.svg","name":"Major","id":"3"},"issuetype":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issuetype/1","id":"1","description":"A problem which impairs or prevents the functions of the product.","iconUrl":"https://eucalyptus.atlassian.net/secure/viewavatar?size=xsmall&avatarId=14803&avatarType=issuetype","name":"Bug","subtask":false,"avatarId":14803}}}}],"assignee":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=tsugawa","name":"tsugawa","key":"tsugawa","accountId":"557058:619bab88-9b87-4778-995a-1e8a85d8b899","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/c16b1da4b6426cad544b6eddd78d76b1?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FavatarId%3D10122%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/c16b1da4b6426cad544b6eddd78d76b1?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26avatarId%3D10122%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/c16b1da4b6426cad544b6eddd78d76b1?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26avatarId%3D10122%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/c16b1da4b6426cad544b6eddd78d76b1?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26avatarId%3D10122%26noRedirect%3Dtrue"},"displayName":"M Tsugawa","active":true,"timeZone":"America/New_York"},"updated":"2016-08-25T10:55:51.290-0500","status":{"self":"https://eucalyptus.atlassian.net/rest/api/2/status/6","description":"The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.","iconUrl":"https://eucalyptus.atlassian.net/images/icons/statuses/generic.png","name":"Closed","id":"6","statusCategory":{"self":"https://eucalyptus.atlassian.net/rest/api/2/statuscategory/3","id":3,"key":"done","colorName":"green","name":"Done"}},"components":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/component/11504","id":"11504","name":"EC2"}],"timeoriginalestimate":null,"customfield_13000":null,"description":"In a VPC setup with 4.2.1 we are facing all of a sudden issue. All our newly launched instances are getting stuck into {{pending}} state. We are seeing the following log entry in nc.log \r\n\r\n{code}\r\n2015-12-20 20:18:35 DEBUG 000037754 instance_network_gate    | [i-82de9636] version (e94a7bae) and applied version (37ea2296) do not match (yet), waiting\r\n2015-12-20 20:18:36 DEBUG 000037754 instance_network_gate    | [i-82de9636] version (e94a7bae) and applied version (37ea2296) do not match (yet), waiting\r\n2015-12-20 20:18:37 DEBUG 000037754 instance_network_gate    | [i-82de9636] version (e94a7bae) and applied version (37ea2296) do not match (yet), waiting\r\n2015-12-20 20:18:38 DEBUG 000037754 instance_network_gate    | [i-82de9636] version (e94a7bae) and applied version (37ea2296) do not match (yet), waiting\r\n2015-12-20 20:18:39 DEBUG 000037754 instance_network_gate    | [i-82de9636] version (e94a7bae) and applied version (37ea2296) do not match (yet), waiting\r\n2015-12-20 20:18:40 DEBUG 000037754 instance_network_gate    | [i-82de9636] version (e94a7bae) and applied version (37ea2296) do not match (yet), waiting\r\n2015-12-20 20:18:41 DEBUG 000037754 instance_network_gate    | [i-82de9636] version (e94a7bae) and applied version (37ea2296) do not match (yet), waiting\r\n{code}\r\n\r\nBased on previous experience we checked the eucanetd.log on the CLC:\r\n\r\n{code}\r\n2015-12-20 20:22:57 DEBUG 000026097 mido_create_rule         | FOUND RULE: { \"type\": \"drop\", \"nwDstAddress\": null, \"nwDstLength\": 0, \"nwSrcAddress\": null, \"nwSrcLength\": 0, \"condInvert\": false, \"matchForwardFlow\": false, \"matchReturnFlow\": false, \"inPorts\": [ ], \"invInPorts\": false, \"outPorts\": [ ], \"invOutPorts\": false, \"portGroup\": null, \"invPortGroup\": false, \"ipAddrGroupSrc\": null, \"invIpAddrGroupSrc\": false, \"ipAddrGroupDst\": null, \"invIpAddrGroupDst\": false, \"traversedDevice\": null, \"invTraversedDevice\": false, \"dlType\": 2054, \"invDlType\": true, \"dlSrc\": null, \"dlSrcMask\": null, \"invDlSrc\": false, \"dlDst\": null, \"dlDstMask\": null, \"invDlDst\": false, \"nwTos\": 0, \"invNwTos\": false, \"nwProto\": 0, \"invNwProto\": false, \"invNwSrc\": false, \"invNwDst\": false, \"fragmentPolicy\": \"any\", \"tpSrc\": null, \"tpDst\": null, \"invTpSrc\": false, \"invTpDst\": false, \"id\": \"142c946a-e5db-483e-8fd2-c1833c717006\", \"chainId\": \"5e038170-d00d-4f00-bb55-3364a7920cb2\", \"meterName\": null, \"position\": 7, \"properties\": { }, \"uri\": \"http:\\/\\/euca2-5:8080\\/midonet-api\\/rules\\/142c946a-e5db-483e-8fd2-c1833c717006\" }\r\n2015-12-20 20:22:57 DEBUG 000026097 mido_create_rule         | FOUND RULE: { \"type\": \"drop\", \"nwDstAddress\": null, \"nwDstLength\": 0, \"nwSrcAddress\": null, \"nwSrcLength\": 0, \"condInvert\": false, \"matchForwardFlow\": false, \"matchReturnFlow\": false, \"inPorts\": [ ], \"invInPorts\": false, \"outPorts\": [ ], \"invOutPorts\": false, \"portGroup\": null, \"invPortGroup\": false, \"ipAddrGroupSrc\": null, \"invIpAddrGroupSrc\": false, \"ipAddrGroupDst\": null, \"invIpAddrGroupDst\": false, \"traversedDevice\": null, \"invTraversedDevice\": false, \"dlType\": 2054, \"invDlType\": true, \"dlSrc\": null, \"dlSrcMask\": null, \"invDlSrc\": false, \"dlDst\": null, \"dlDstMask\": null, \"invDlDst\": false, \"nwTos\": 0, \"invNwTos\": false, \"nwProto\": 0, \"invNwProto\": false, \"invNwSrc\": false, \"invNwDst\": false, \"fragmentPolicy\": \"any\", \"tpSrc\": null, \"tpDst\": null, \"invTpSrc\": false, \"invTpDst\": false, \"id\": \"9ce8f5b3-0b68-4d82-9929-54a200deef91\", \"chainId\": \"59217d1b-d16f-41b1-9ec0-8744a018a3cd\", \"meterName\": null, \"position\": 4, \"properties\": { }, \"uri\": \"http:\\/\\/euca2-5:8080\\/midonet-api\\/rules\\/9ce8f5b3-0b68-4d82-9929-54a200deef91\" }\r\n2015-12-20 20:22:57 ERROR 000026097 do_midonet_update        | pass3_insts: failed update - check midonet health\r\n2015-12-20 20:22:57 ERROR 000026097 network_driver_system_sc | could not update midonet: check log for details\r\n2015-12-20 20:22:57 ERROR 000026097 main                     | could not complete VM network update: check above log errors for details\r\n{code}\r\n\r\nWe are able to use {{midonet-cli}} to interact with the midonet system but there appears to be a problem. The setup is currently like the following:\r\n\r\na) CLC+Midolman+Eucanetd - 1 server\r\nb) CC/SC for availability zone 1 + ZK + CS + Midolman - 1 server\r\nc) CC/SC for availability zone 2 - 1 server\r\nd) NC x 2 - 2 servers for AZ-01\r\ne) NC x 1 - 1 server for AZ-02\r\nf) UFS + Console - 1 server\r\n\r\nOn the CS host we can clearly see errors:\r\n\r\n{code}\r\nuested, slices=[-]\r\n WARN [ReadStage:92704] 2015-12-20 20:22:19,246 SliceQueryFilter.java (line 296) Read 44 live and 28189 tombstone cells in midonetflowstate.conntrack_by_egress_port.conntrack_by_egress_port_port_idx (see tombstone_warn_threshold). 5000 columns was requested, slices=[-]\r\nERROR [ReadStage:92703] 2015-12-20 20:22:19,253 SliceQueryFilter.java (line 271) Scanned over 100000 tombstones in midonetflowstate.conntrack_by_egress_port.conntrack_by_egress_port_port_idx; query aborted (see tombstone_failure_threshold)\r\nERROR [ReadStage:92703] 2015-12-20 20:22:19,253 CassandraDaemon.java (line 258) Exception in thread Thread[ReadStage:92703,5,main]\r\njava.lang.RuntimeException: org.apache.cassandra.db.filter.TombstoneOverwhelmingException\r\n\tat org.apache.cassandra.service.StorageProxy$DroppableRunnable.run(StorageProxy.java:2013)\r\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)\r\n\tat java.lang.Thread.run(Thread.java:745)\r\nCaused by: org.apache.cassandra.db.filter.TombstoneOverwhelmingException\r\n\tat org.apache.cassandra.db.filter.SliceQueryFilter.collectReducedColumns(SliceQueryFilter.java:273)\r\n\tat org.apache.cassandra.db.filter.QueryFilter.collateColumns(QueryFilter.java:122)\r\n\tat org.apache.cassandra.db.filter.QueryFilter.collateOnDiskAtom(QueryFilter.java:80)\r\n\tat org.apache.cassandra.db.filter.QueryFilter.collateOnDiskAtom(QueryFilter.java:72)\r\n\tat org.apache.cassandra.db.CollationController.collectAllData(CollationController.java:297)\r\n\tat org.apache.cassandra.db.CollationController.getTopLevelColumns(CollationController.java:53)\r\n\tat org.apache.cassandra.db.ColumnFamilyStore.getTopLevelColumns(ColumnFamilyStore.java:1619)\r\n\tat org.apache.cassandra.db.ColumnFamilyStore.getColumnFamily(ColumnFamilyStore.java:1438)\r\n\tat org.apache.cassandra.db.index.composites.CompositesSearcher$1.computeNext(CompositesSearcher.java:163)\r\n\tat org.apache.cassandra.db.index.composites.CompositesSearcher$1.computeNext(CompositesSearcher.java:103)\r\n\tat com.google.common.collect.AbstractIterator.tryToComputeNext(AbstractIterator.java:143)\r\n\tat com.google.common.collect.AbstractIterator.hasNext(AbstractIterator.java:138)\r\n\tat org.apache.cassandra.db.ColumnFamilyStore.filter(ColumnFamilyStore.java:1825)\r\n\tat org.apache.cassandra.db.index.composites.CompositesSearcher.search(CompositesSearcher.java:50)\r\n\tat org.apache.cassandra.db.index.SecondaryIndexManager.search(SecondaryIndexManager.java:552)\r\n\tat org.apache.cassandra.db.ColumnFamilyStore.search(ColumnFamilyStore.java:1812)\r\n\tat org.apache.cassandra.db.PagedRangeCommand.executeLocally(PagedRangeCommand.java:113)\r\n\tat org.apache.cassandra.service.StorageProxy$LocalRangeSliceRunnable.runMayThrow(StorageProxy.java:1466)\r\n\tat org.apache.cassandra.service.StorageProxy$DroppableRunnable.run(StorageProxy.java:2009)\r\n\t... 3 more\r\n WARN [ReadStage:92681] 2015-12-20 20:22:19,292 SliceQueryFilter.java (line 296) Read 88 live and 55322 tombstone cells in midonetflowstate.nat_by_ingress_port.nat_by_ingress_port_port_idx (see tombstone_warn_threshold). 5000 columns was requested, slices=[-]\r\n{code}\r\n\r\nThe CS logs are getting extremely quickly populated with the above messages. Looks like some cleanup is required in CS for egress.\r\n\r\nThe midolman logs on the same host also suggest lot of errors:\r\n\r\n{code}\r\n2015.12.20 20:21:02.348 WARN  [midolman-akka.actor.default-dispatcher-5] host-proxy -  Failed to fetch state\r\njava.util.concurrent.TimeoutException: null\r\n\tat org.midonet.midolman.state.FlowStateStorageImpl$$anonfun$1.apply$mcV$sp(FlowStateStorage.scala:349) ~[midolman-1.9.6.jar:1.9.6]\r\n\tat akka.actor.Scheduler$$anon$7.run(Scheduler.scala:117) ~[akka-actor_2.11-2.3.6.jar:na]\r\n\tat akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:41) ~[akka-actor_2.11-2.3.6.jar:na]\r\n\tat akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:393) [akka-actor_2.11-2.3.6.jar:na]\r\n\tat scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library-2.11.2.jar:na]\r\n\tat scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library-2.11.2.jar:na]\r\n\tat scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library-2.11.2.jar:na]\r\n\tat scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library-2.11.2.jar:na]\r\n2015.12.20 20:21:09.332 WARN  [midonet-nio-worker-3] FlowStateStorage -  failed to complete future with failure {}\r\ncom.datastax.driver.core.exceptions.ReadTimeoutException: Cassandra timeout during read query at consistency QUORUM (1 responses were required but only 0 replica responded)\r\n\tat com.datastax.driver.core.Responses$Error$1.decode(Responses.java:56) ~[cassandra-driver-core-2.0.10.jar:na]\r\n\tat com.datastax.driver.core.Responses$Error$1.decode(Responses.java:33) ~[cassandra-driver-core-2.0.10.jar:na]\r\n\tat com.datastax.driver.core.Message$ProtocolDecoder.decode(Message.java:208) ~[cassandra-driver-core-2.0.10.jar:na]\r\n\tat com.datastax.driver.core.Message$ProtocolDecoder.decode(Message.java:199) ~[cassandra-driver-core-2.0.10.jar:na]\r\n\tat io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:89) [netty-codec-4.0.27.Final.jar:4.0.27.Final]\r\n\t... 11 common frames omitted\r\nWrapped by: com.datastax.driver.core.exceptions.ReadTimeoutException: Cassandra timeout during read query at consistency QUORUM (1 responses were required but only 0 replica responded)\r\n\tat com.datastax.driver.core.exceptions.ReadTimeoutException.copy(ReadTimeoutException.java:69) ~[cassandra-driver-core-2.0.10.jar:na]\r\n\tat com.datastax.driver.core.Responses$Error.asException(Responses.java:93) ~[cassandra-driver-core-2.0.10.jar:na]\r\n\tat com.datastax.driver.core.DefaultResultSetFuture.onSet(DefaultResultSetFuture.java:114) [cassandra-driver-core-2.0.10.jar:na]\r\n\tat com.datastax.driver.core.RequestHandler.setFinalResult(RequestHandler.java:183) [cassandra-driver-core-2.0.10.jar:na]\r\n\tat com.datastax.driver.core.RequestHandler.access$2300(RequestHandler.java:45) [cassandra-driver-core-2.0.10.jar:na]\r\n\tat com.datastax.driver.core.RequestHandler$SpeculativeExecution.setFinalResult(RequestHandler.java:746) [cassandra-driver-core-2.0.10.jar:na]\r\n\tat com.datastax.driver.core.RequestHandler$SpeculativeExecution.onSet(RequestHandler.java:585) [cassandra-driver-core-2.0.10.jar:na]\r\n\tat com.datastax.driver.core.Connection$Dispatcher.channelRead0(Connection.java:984) [cassandra-driver-core-2.0.10.jar:na]\r\n\tat com.datastax.driver.core.Connection$Dispatcher.channelRead0(Connection.java:917) [cassandra-driver-core-2.0.10.jar:na]\r\n\tat io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:105) [netty-transport-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:339) [netty-transport-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:324) [netty-transport-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:254) [netty-all-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:339) [netty-transport-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:324) [netty-transport-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) [netty-codec-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:339) [netty-transport-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:324) [netty-transport-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:242) [netty-codec-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:339) [netty-transport-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:324) [netty-transport-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:847) [netty-transport-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:618) [netty-all-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:329) [netty-all-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:250) [netty-all-4.0.27.Final.jar:4.0.27.Final]\r\n\tat io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:111) [netty-all-4.0.27.Final.jar:4.0.27.Final]\r\n\tat java.lang.Thread.run(Thread.java:745) [na:1.7.0_91]\r\n2015.12.20 20:21:11.056 WARN  [midolman-akka.actor.default-dispatcher-5] host-proxy -  Failed to fetch state\r\njava.util.concurrent.TimeoutException: null\r\n\tat org.midonet.midolman.state.FlowStateStorageImpl$$anonfun$1.apply$mcV$sp(FlowStateStorage.scala:349) ~[midolman-1.9.6.jar:1.9.6]\r\n\tat akka.actor.Scheduler$$anon$7.run(Scheduler.scala:117) ~[akka-actor_2.11-2.3.6.jar:na]\r\n\tat akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:41) ~[akka-actor_2.11-2.3.6.jar:na]\r\n\tat akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:393) [akka-actor_2.11-2.3.6.jar:na]\r\n\tat scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library-2.11.2.jar:na]\r\n\tat scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library-2.11.2.jar:na]\r\n\tat scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library-2.11.2.jar:na]\r\n\tat scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library-2.11.2.jar:na]\r\n{code}\r\n\r\nWith this state we do not know how to recover back to normal. The system has entered into this state on its own without any manual configuration or process related change done by the admin. Currently we do not have many instances running on the cloud (16 instances and 13 security groups).\r\n\r\n","customfield_13200":null,"customfield_11300":null,"customfield_11500":null,"timetracking":{},"customfield_10401":null,"customfield_10005":null,"customfield_10600":null,"customfield_12900":null,"attachment":[],"aggregatetimeestimate":null,"summary":"Instance getting stuck in pending","creator":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"subtasks":[],"reporter":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"aggregateprogress":{"progress":0,"total":0},"customfield_12100":"0|i03k3z:","customfield_10000":"5_*:*_1_*:*_0_*|*_10000_*:*_1_*:*_1543154233_*|*_10001_*:*_1_*:*_4847300513","customfield_10001":"2016-03-03T18:41:15.217-0600","customfield_10200":null,"customfield_10002":null,"customfield_10003":null,"customfield_10400":null,"customfield_12500":null,"customfield_10116":null,"customfield_11600":["dshekhawat(dshekhawat)","dmitrii(dmitrii)","jgallucci32(jgallucci32)","tsugawa(tsugawa)"],"environment":null,"customfield_10117":null,"customfield_11800":null,"customfield_11802":null,"duedate":null,"customfield_11805":null,"progress":{"progress":0,"total":0},"customfield_11806":null,"votes":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/EUCA-11852/votes","votes":0,"hasVoted":false},"comment":{"comments":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40044/comment/97228","id":"97228","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"body":"The environment is currently used for a POC. It is hosted at Alpharetta. ","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"created":"2015-12-20T19:35:35.809-0600","updated":"2015-12-20T19:35:35.809-0600"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40044/comment/97301","id":"97301","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"body":"After we hit this issue we just started running new instances via CloudFormation and so far have not hit the problem however CS continues to spew out messages in the log files etc. The midonet state however has improved and eucanetd does not report any errors on its health.","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"created":"2015-12-21T05:07:38.084-0600","updated":"2015-12-21T05:07:38.084-0600"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40044/comment/100005","id":"100005","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dmitrii","name":"dmitrii","key":"dmitrii","accountId":"557058:1e0ea807-434b-4b54-967f-1dec4a9ede55","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/efb56b507e22d4fc0d8d5e900e5651d5?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddmitrii%26avatarId%3D15900%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/efb56b507e22d4fc0d8d5e900e5651d5?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddmitrii%26avatarId%3D15900%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/efb56b507e22d4fc0d8d5e900e5651d5?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddmitrii%26avatarId%3D15900%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/efb56b507e22d4fc0d8d5e900e5651d5?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddmitrii%26avatarId%3D15900%26noRedirect%3Dtrue"},"displayName":"Dmitrii Calzago","active":false,"timeZone":"America/Los_Angeles"},"body":"It appears that the problem was spurious. Please, reopen if this problem surfaces again.","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dmitrii","name":"dmitrii","key":"dmitrii","accountId":"557058:1e0ea807-434b-4b54-967f-1dec4a9ede55","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/efb56b507e22d4fc0d8d5e900e5651d5?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddmitrii%26avatarId%3D15900%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/efb56b507e22d4fc0d8d5e900e5651d5?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddmitrii%26avatarId%3D15900%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/efb56b507e22d4fc0d8d5e900e5651d5?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddmitrii%26avatarId%3D15900%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/efb56b507e22d4fc0d8d5e900e5651d5?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddmitrii%26avatarId%3D15900%26noRedirect%3Dtrue"},"displayName":"Dmitrii Calzago","active":false,"timeZone":"America/Los_Angeles"},"created":"2016-03-03T18:41:15.217-0600","updated":"2016-03-03T18:41:15.217-0600"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40044/comment/109506","id":"109506","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=jgallucci32","name":"jgallucci32","key":"jgallucci32","accountId":"557058:1cd98ee4-b2ee-4683-a315-2a6abd75b048","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FavatarId%3D10122%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26avatarId%3D10122%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26avatarId%3D10122%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26avatarId%3D10122%26noRedirect%3Dtrue"},"displayName":"John Gallucci","active":true,"timeZone":"America/Los_Angeles"},"body":"We have been experiencing this issue ever since I stood up a new 4.2.1 environment with VPC. It takes 2-3 minutes to create any instance in our cloud. We are seeing the following error messages repeatedly on the NC nodes until the instance is in the running state:\r\n{noformat}\r\n2016-08-25 07:23:45 DEBUG 000027526 instance_network_gate    | [i-f57a6c2b] version (c05be5c5) and applied version (50cc9b3b) do not match (yet), waiting\r\n{noformat}\r\nEven after an upgrade to 4.2.2 we are still experiencing this issue. I will continue to investigate why Midonet is causing a delay here.","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=jgallucci32","name":"jgallucci32","key":"jgallucci32","accountId":"557058:1cd98ee4-b2ee-4683-a315-2a6abd75b048","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FavatarId%3D10122%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26avatarId%3D10122%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26avatarId%3D10122%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26avatarId%3D10122%26noRedirect%3Dtrue"},"displayName":"John Gallucci","active":true,"timeZone":"America/Los_Angeles"},"created":"2016-08-25T09:30:16.430-0500","updated":"2016-08-25T09:30:16.430-0500"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40044/comment/109508","id":"109508","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=jgallucci32","name":"jgallucci32","key":"jgallucci32","accountId":"557058:1cd98ee4-b2ee-4683-a315-2a6abd75b048","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FavatarId%3D10122%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26avatarId%3D10122%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26avatarId%3D10122%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26avatarId%3D10122%26noRedirect%3Dtrue"},"displayName":"John Gallucci","active":true,"timeZone":"America/Los_Angeles"},"body":"One thing I noticed is that in euanetd.log on the CLC, it is scrubbing the entire VPCMIDO database every minute.  This appears in the log every minute:\r\n{noformat}\r\n2016-08-25 08:49:28  INFO | new Eucalyptus/Midonet networking state sync: updated successfully\r\n2016-08-25 08:49:29  INFO | update triggered due to file update (//var/run/eucalyptus/eucanetd_global_network_info.xml)\r\n2016-08-25 08:49:29  INFO | new networking state (VM network/security groups/addressing): updating system\r\n2016-08-25 08:49:29  INFO | Scrubbing for 'VPCMIDO' network driver.\r\n{noformat}\r\nThe time it takes to scrub the entire VPCMIDO environment takes 1-2 minutes, the same time it takes for a new instance to be created.\r\n\r\nIs it normal for the network sync to occur every minutes?\r\n\r\nAnother anomoly I should note is we had our midonet logs getting full because of the STDOUT from the Midonet API.  I wonder if something is stuck here.","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=jgallucci32","name":"jgallucci32","key":"jgallucci32","accountId":"557058:1cd98ee4-b2ee-4683-a315-2a6abd75b048","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FavatarId%3D10122%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26avatarId%3D10122%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26avatarId%3D10122%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/31cfdf964a6ac26962c22f3279e2c85c?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26avatarId%3D10122%26noRedirect%3Dtrue"},"displayName":"John Gallucci","active":true,"timeZone":"America/Los_Angeles"},"created":"2016-08-25T10:55:51.290-0500","updated":"2016-08-25T10:55:51.290-0500"}],"maxResults":5,"total":5,"startAt":0},"worklog":{"startAt":0,"maxResults":20,"total":0,"worklogs":[]},"customfield_11808":null}}